bedrock/firefox/templates/firefox/recommended.html

```diff
@@ -0,0 +1,92 @@
+    <div class="mzp-l-content">
+      <div class="mzp-c-hero-body">
+        <h1 class="mzp-c-hero-title">{{ _('Fast. Private. Fearless. <br>Firefox fights for you.') }}</h1>
+
+        <div class="mzp-c-hero-cta mzp-c-button-download">
+          {{ download_firefox(alt_copy=_('Download Firefox'), download_location='primary cta', button_color='mzp-c-button mzp-t-download') }}
+        </div>
+
+        <div class="mzp-c-hero-desc">
+          <p>{{ _('This browser comes highly recommended.') }}</p>
+          <p>{{ _('You know someone who knows how the web works. And they know that the web works best on Firefox.') }}</p>
+        </div>
+      </div>
+
+      {{ high_res_img('firefox/recommended/hero-laptop.png', {'alt': '', 'width': '672', 'height': '402'}) }}
+    </div>
+  </section>
+
+  <div class="mzp-l-card-third mzp-l-content mzp-t-firefox">
+    <ul>
+      <li class="faster">
+        <section class="mzp-c-card-picto">
+          <div class="mzp-c-card-picto-content">
+            <h2 class="mzp-c-card-picto-title">{{_('2x Faster') }}</h2>
+            <p class="mzp-c-card-picto-desc">{{_('The best Firefox ever') }}</p>
+          </div>
+        </section>
+      </li>
+      <li class="lighter">
+        <section class="mzp-c-card-picto">
+          <div class="mzp-c-card-picto-content">
+            <h2 class="mzp-c-card-picto-title">{{_('Lightweight') }}</h2>
+            <p class="mzp-c-card-picto-desc">{{_('Rebuilt to use less memory') }}</p>
+          </div>
+        </section>
+      </li>
+      <li class="private">
+        <section class="mzp-c-card-picto">
+          <div class="mzp-c-card-picto-content">
+            <h2 class="mzp-c-card-picto-title">{{_('Powerfully private') }}</h2>
+            <p class="mzp-c-card-picto-desc">{{_('Truly Private Browsing with Tracking Protection') }}</p>
+          </div>
+        </section>
+      </li>
+    </ul>
+  </div>
+
+  <aside class="c-aside">
+    <div class="mzp-l-content">
+      <p><a href="https://mozilla.github.io/for-firefox/" data-link-type="link" data-link-name="Want a Firefox badge of your own?">{{ _('Want a Firefox badge of your own?') }}</a></p>
+    </div>
+  </aside>
+
+  <section class="secondary-cta mzp-c-call-out-compact mzp-t-firefox mzp-t-product-firefox">
+      <div class="mzp-l-content">
+        <div class="mzp-c-call-out-content">
+          <div class="mzp-c-call-out-container">
+          <h2 class="mzp-c-call-out-title">{{ _('Take control of your browser.') }}</h2>
+          </div>
+        </div>
+        <div class="mzp-c-call-out-cta">
+          <div class="mzp-c-call-out-cta-container">
+            <div class="mzp-c-button-download">
+              {{ download_firefox(alt_copy=_('Download Firefox'), download_location='secondary cta', button_color='mzp-c-button mzp-t-download') }}
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
```

Comment:
Maybe center this content for `< 768px` so it matches the rest of the page?

Reply:
This content is also a little misaligned in IE 10 & 11. (But maybe not enough to worry too much about?)

<img width="1051" alt="screen shot 2018-10-05 at 10 25 48 am" src="https://user-images.githubusercontent.com/317532/46544741-fe75ae00-c889-11e8-96a6-c5d3e659967d.png">

Reply:
This is a protocol component / bug, which is being tracked upstream https://github.com/mozilla/protocol/issues/218, so not fixing here.